crates/light-client-prover/src/tests/mod.rs

```diff
@@ -2157,3 +2159,207 @@ fn wrong_pubkey_sequencer_commitment_should_not_work() {
     // ignored commitment from wrong pubkey
     assert_eq!(commitment, None);
 }
+
+// For some reason, even though macro is used, it sees it as unused
+#[allow(unused)]
+macro_rules! assert_panic {
+    // Match a single expression
```

Comment:
I believe its because macro definitions should actually lie in lib.rs of the crate, and there is some weird rules around defining macros except there. Thats why a lot of external crates have separate crate for defining macros.